src/styles/prism.js

```diff
@@ -0,0 +1,144 @@
+import { css } from "styled-components";
+
```

Comment:
I think you may be able to include this from the prism js package instead

Reply:
but would we be able to use the custom colors from `theme` if we did that? do they have a way to do the theming?

and if we just use this file, do we need the prism dep?